psi4/src/psi4/libfock/jk.h

```diff
@@ -938,5 +939,85 @@ class CDJK : public DFJK {
     /// Destructor
     virtual ~CDJK();
 };
+
+/**
+ * Class symm_JK
+ *
+ * JK implementation using sieved, threaded
+ * density-fitted technology
+ * under slightly different paradigm than DFJK
+ * wraps lib3index/DF_Helper class
+ */
+class symm_JK : public JK {
```

Comment:
Could we call this guy `symm_DFJK` so that we are clear that this is a flavor of DFJK.